rest-api/model/participant_summary.py

```diff
@@ -47,8 +47,8 @@ class ParticipantSummary(Base):
   phoneNumber = Column('phone_number', String(80))
   loginPhoneNumber = Column('login_phone_number', String(80))
   email = Column('email', String(255))
+  primaryLanguage = Column('primary_language', String(80))
   recontactMethodId = Column('recontact_method_id', Integer, ForeignKey('code.code_id'))
-  languageId = Column('language_id', Integer, ForeignKey('code.code_id'))
```

Comment:
So I believe removing this field will change our responses so that instead of having "language": "UNSET" for every participant in prod, there simply won't be a language field. Which is technically a breaking change; we need clients to migrate away from using language before we can get rid of it entirely.

The simplest thing is to keep this field around until clients have migrated to use primaryLanguage, and then get rid of it. Alternatively, you change participant_summary_dao.to_client_json to always set language to "UNSET".